react-app/src/app/screens/Game/components/Moves/components/Move/index.js

```diff
@@ -8,21 +8,32 @@ import calculateWinner from '../../../../utils';
 import style from './styles.scss';
 
 class Move extends Component {
+  state = {
+    history: this.props.history,
+    squares: [],
+    winner: null,
+    desc: this.props.move ? `Go to move # ${this.props.move}` : `Go to game start`
```

Comment:
You are assigning a constant value to the description. It wont change since this is only calculated on mount. This logic should be done in getDerivedStateFromProps